### ISSUE TITLE
Background Color

### ISSUE THREAD

**Issue by crocis** [aipixel/GPS-Gaussian#43]
I have trained GPS-Gaussian with black background color (in stage2 I use the default value of bg_color as defined in stereo_human_config.py), and then tested with black and white background colors. I have noticed, that when I run GPS-Gaussian with different background colors, I get different foreground colors. The rendered person is brighter with white color than with black color. It seems that some Gaussians are transparent or there are gaps between Gaussians, so that the background color is also visible in the foreground.

Here is a rendered person with black and white background colors. 
![0130_novel_black](https://github.com/aipixel/GPS-Gaussian/assets/2960536/8b3875d0-f61c-4b47-b215-dad4ed9401e5)
![0130_novel_white](https://github.com/aipixel/GPS-Gaussian/assets/2960536/32150980-93b7-4bc9-a95d-ccbb978b857e)

Do I really need to train GPS-Gaussian with the same background color that is used at inference? Why?

**Comment by ShunyuanZheng** [comment]
I have also come up with the same problem when I want to render a novel view on a white background with a model pretrained on black background. I trained another model with white background images to solve this. However, I have not delved into this issue.

**Comment by crocis** [comment]
I am training GPS-Gaussian randomizing the background color, so that the network won't assume a constant background color. So far, PSNR metric applied on the validation set is bad. Are there other ways to train GPS-Gaussian so that it does not assume a constant background color?
